src/webui/api/torrentscontroller.cpp

```diff
@@ -320,10 +406,22 @@ void TorrentsController::trackersAction()
         case BitTorrent::TrackerEntry::NotWorking:
             status = tr("Not working"); break;
         }
+        trackerDict[KEY_TRACKER_TIER] = tracker.tier();
         trackerDict[KEY_TRACKER_STATUS] = status;
-        trackerDict[KEY_TRACKER_PEERS] = data.numPeers;
+        trackerDict[KEY_TRACKER_DEPRECATE_PEERS] = data.numPeers;
+        trackerDict[KEY_TRACKER_TRACKER_PEERS_COUNT] = data.numPeers;
         trackerDict[KEY_TRACKER_MSG] = data.lastMessage.trimmed();
 
+#if LIBTORRENT_VERSION_NUM >= 10000
+        trackerDict[KEY_TRACKER_SEEDS_COUNT] = (tracker.nativeEntry().scrape_complete > 0) ? tracker.nativeEntry().scrape_complete : 0;
+        trackerDict[KEY_TRACKER_PEERS_COUNT] = (tracker.nativeEntry().scrape_incomplete > 0) ? tracker.nativeEntry().scrape_incomplete : 0;
+        trackerDict[KEY_TRACKER_DOWNLOADED_COUNT] = (tracker.nativeEntry().scrape_downloaded > 0) ? tracker.nativeEntry().scrape_downloaded : 0;
```

Comment:
We changed these in e2b46f01bb19b7faba0f4b3dd050115562ac16c3

Reply:
I've implemented this change client side (in `prop-trackers.js`). It seems more reasonable to me that this API would return numbers rather than strings, but am open to discuss if you feel strongly.

Reply:
I suppose this still doesn't disambiguate an unconnected tracker from a connected tracker with 0 seeds/peers. I'll move the change to the API.